Project21/Project21/conversionfile.c

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.Naveenkumar s
+// ------------------------------------------------------------------
+// Assignment 3
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#define INT_SIZE sizeof(int)*8

```

Comment:
Usage of expressions in #define causes bugs. It is better to avoid this. You can compute this in your existing functions, or declare a new function for it.